sources/packages/web/src/views/institution/Login.vue

```diff
@@ -31,17 +36,21 @@
 
 <script lang="ts">
 import { AppConfigService } from "../../services/AppConfigService";
+import { useRoute } from "vue-router";
 
 export default {
   components: {},
-  setup() {
+
+  setup(props: any) {
+    const route = useRoute();
+    const basicBCeID = route.query.basicBCeID;
```

Comment:
This mapping could be done directly into the routes enabling the "props: true". In this way, no additional code is needed here. In the example below the ":id?" is an optional parameter that will be mapped to an "id" property in the component.

```suggestion
{
        path: "dynamic-application-form/:id?",
        name: StudentRoutesConst.DYNAMIC_FINANCIAL_APP_FORM,
        component: DynamicStudentApp,
        props: true,
      },
```

Reply:
In the interest of time, Can you approve it so that i can merge this PR and i can submit another PR for this small change?

Reply:
We were not able to do it right now.